torch/ao/quantization/quantizer/xnnpack_quantizer_utils.py

```diff
@@ -52,6 +52,23 @@ class QuantizationConfig:
     "torch.ao.quantization.quantizer.xnnpack_quantizer_utils"
 )
 
+AnnotatorType = Callable[
+    [
+        torch.fx.GraphModule,
+        Optional[QuantizationConfig],
+        Optional[Callable[[Node], bool]],
+    ],
+    Optional[List[List[Node]]],
+]
+OP_TO_ANNOTATOR: Dict[str, AnnotatorType] = {}
+
+
+def register_annotator(op: str):
+    def decorator(annotator: AnnotatorType):
+        OP_TO_ANNOTATOR[op] = annotator
```

Comment:
nit: I think it would make sense to add `annotate` function here that looks up the string in the dict.

Reply:
sorry what do you mean? could you write a small example?

Reply:
sorry, misplaced comments. What i meant was how it is used in xnnpack_quantizer.

```
def annotate(op: str, qconfig,...):
  return OP_TO_ANNOTATOR[op](qconfig,....)

from xnnpack_quantizer_utils import annotate

....
def _annotate_conv2d_relu(..)
   annotate("conv2d_relu",...)
```

Reply:
oh OK I can refactor it in next PR